packages/combo-box/test/basic.test.js

```diff
@@ -31,7 +31,7 @@ describe('Properties', () => {
 
       comboBox.items = ['foo', 'bar'];
 
-      expect(comboBox._focusedIndex).to.eql(1);
+      expect(comboBox._focusedIndex).to.eql(-1);
```

Comment:
This test doesn't make much sense to me after removing related logic. Should we remove it?

Reply:
Done.